CMake/External_OpenCV.cmake

```diff
@@ -82,6 +82,21 @@ else()
 endif()
 
 
+if(fletch_ENABLE_OpenBLAS)
+  if (OpenCV_version VERSION_LESS 3.2.0)
+  else()
+    message(STATUS "OpenCV depending on fletch OpenBLAS")
+    set(_OpenCV_ENABLE_OPENBLAS_DEFAULT TRUE)
+    #list(APPEND OpenCV_DEPENDS OpenBLAS)  # complains that this target doesnt exist. Does it?
+    list(APPEND OpenCV_EXTRA_BUILD_FLAGS
+      -DWITH_LAPACK:BOOL=FALSE  # workaround for undefined referenec to `gotoblas`
+      -DOpenBLAS_INCLUDE_DIR:PATH="${OpenBLAS_ROOT}/include"
+      -DOpenBLAS_LIB:PATH="${OpenBLAS_ROOT}/lib/${openblas_libname}"
```

Comment:
I don't see openblas_libname defined in this file. I suppose if it's working, we might be getting lucky and picking it up from the Caffe configuration. Better to have it in here in case someone only builds openBLAS and OpenCV.